test/support/meetups/sample_api_responses.rb

```diff
@@ -0,0 +1,127 @@
+require 'ostruct'
```

Comment:
Ruby's [OpenStruct](https://ruby-doc.org/stdlib-2.3.0/libdoc/ostruct/rdoc/OpenStruct.html) is a cool tool to have in you toolbox.

Reason we need this here is because it allows us to create an object, and then add methods to it that we can call using `.` notation.

So for example:
- we create an (OpenStruct) object called `response_struct`
- we then add a method/attribute to it called `code`
- we set this `code` attribute equal to `200` (by default)

This allows us to call what is in the `code` attribute using `.` notation:

```ruby
[2] pry(main)> response_struct = OpenStruct.new
=> #<OpenStruct>
[3] pry(main)> response_struct.code = 200
=> 200
[4] pry(main)> response_struct.code
=> 200
```

And this is exactly how the response returned from the Meetup API is structured.  You are checking that it's code is 200 by calling `response.code`.  `OpenStruc` allows for this.